tool/pcore/src/main/java/org/openscience/cdk/pharmacophore/PharmacophoreMatcher.java

```diff
@@ -426,8 +426,8 @@ private IAtomContainer getPharmacophoreMolecule(IAtomContainer input) throws CDK
             for (IBond bond : pharmacophoreQuery.bonds()) {
                 if (!(bond instanceof PharmacophoreQueryAngleBond)) continue;
 
-                IAtom startQAtom = bond.getBegin();
-                IAtom middleQAtom = bond.getEnd();
+                IAtom startQAtom = bond.getAtom(0);
+                IAtom middleQAtom = bond.getAtom(1);
```

Comment:
Because ``bond.getAtom(2)`` is called after so it's slightly neater. I don't mind using the 0, 1 if there is a 2 (ofcourse I'd remove all bonds with more than two atoms if I could)